### ISSUE TITLE
Connection refused while connecting to upstream

### ISSUE THREAD

**Issue by crunchtime-ali** [pl3bs/dontdockmebro#6]
I've installed Discourse using your script as described. After rebooting I get `502 - Bad Gateway`. When looking into nginx' error log after trying to access the page via the browser I find this error:

`[error] 335#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 36.46.43.32, server: _, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "example.com"`

The host is ovh.com and I used the Ubuntu 15.04 installation-option.

What might be the cause for this problem?

**Comment by pl3bs** [comment]
There's nothing wrong with the script, it was tested by another user just two days prior.

94: read -p "Enter the name of your domain [ex: www.webeindustry.com] " domain;
95: sed -i "s/"www.example.com"/$domain/g" discourse.conf;
It seems you've skipped this part.

**Comment by crunchtime-ali** [comment]
I really had the wrong domain set up but I changed it the `HOSTNAME` in `/var/www/discourse/config/discourse.conf` rebooted but still get the same error.

**Comment by ghost** [comment]
Yeah for some reason, if you type the wrong hostname and try to fix it, it wont work. Just reinstall.
